std_msgs/Float32MultiArray flat_state
uint8 ndimensions
uint8 nderivatives
